directanswercards/allfields-standard/component.js

```diff
@@ -103,10 +103,21 @@ class allfields_standardComponent extends BaseDirectAnswerCard['allfields-standa
         value = isArray ? arrayValue : regularValue;
         break;
       case 'hours':
+        const timezoneOffsetForLocation = relatedItem?.data?.fieldValues?.timeZoneUtcOffset;
```

Comment:
This is some pretty snazzy syntactic sugar.

Reply:
haha yes! just double checked that it works on ie11 thanks to babel

Reply:
I agree. It's probably a good idea for us to use optional chaining more frequently in our javascript